client/webpack.common.config.js

```diff
@@ -6,12 +6,8 @@ module.exports = {
 
   // the project dir
   context: __dirname,
-  entry: ['./assets/javascripts/App'],
+  entry: ['jquery', 'jquery-ujs', './assets/javascripts/App'],
```

Comment:
that's a neat option

I didn't know about that

let's go with that if it works!